fdbclient/S3Client.actor.cpp

```diff
@@ -494,15 +500,26 @@ ACTOR static Future<Void> copyDownFile(Reference<S3BlobStoreEndpoint> endpoint,
 		parts.reserve(numParts);
 		downloadFutures.reserve(numParts);
 		Reference<IAsyncFile> f = wait(IAsyncFileSystem::filesystem()->open(
-		    filepath, IAsyncFile::OPEN_CREATE | IAsyncFile::OPEN_READWRITE | IAsyncFile::OPEN_UNCACHED, 0644));
+		    filepath,
+		    IAsyncFile::OPEN_CREATE | IAsyncFile::OPEN_READWRITE | IAsyncFile::OPEN_UNCACHED |
+		        IAsyncFile::OPEN_ATOMIC_WRITE_AND_CREATE | IAsyncFile::OPEN_NO_AIO,
```

Comment:
Why do we add those new flags?

Reply:
Simulation asserts trip if we don't add OPEN_NON_AIO and OPEN_ATOMIC_WRITE_AND_CREATE

Reply:
Can you run a kube test with the new flags to confirm it works well in a real cluster? Thanks!